Active Documents/contenteditable-disabled.html

```diff
@@ -0,0 +1,192 @@
+        </p>
+        <p>
+            WYSIWYG markdown editors provide a canonical example. While markdown editors typically support bold or italicized text, they often don't support font color or font family customization. System or browser UI elements like those shown for iOS or the Mac Touch Bar below don't have any indication as to which formatting commands are applicable to a particular contenteditable element and should be displayed, and which ones should be disabled or omitted.
+        </p>
+        <figure>
+            <img src="images/mac-touchbar.png" alt="Mac Touch Bar showing editing buttons">
+            <figcaption>
+                Mac Touch Bar showing editing buttons
+            </figcaption>
+        </figure>
+        <figure>
+            <img src="images/ios-editing-buttons.png" width="300" alt="iOS selection menu showing editing buttons">
+            <figcaption>
+                iOS selection menu showing editing buttons
+            </figcaption>
+        </figure>
+        <p>
+            To address the issue, a new contenteditable-disabled attribute can be applied as shown in the example below.
+        </p>
+        <pre class="example">
+            &lt;div contenteditable="true" contenteditable-disabled="
+                formatbackcolor
+                formatdent
+                formatfont
+                formatjustify
+                formatscript
+                formatstrikethrough
+                formatunderline
+                inserthorizontalrule">
+                This is a contenteditable region of a document that supports:
+                &lt;ul>
+                    &lt;li>bold text&lt;/li>
+                    &lt;li>italic text&lt;/li>
+                    &lt;li>links&lt;/li>
+                    &lt;li>lists&lt;/li>
+                    &lt;li>and possibly other formatting outside the scope of this specification.&lt;/li>
+                &lt;/ul>
+                
+                This contenteditable region explicitly does not support:
+                &lt;ul>
+                    &lt;li>background colors&lt;/li>
+                    &lt;li>indentation&lt;/li>
+                    &lt;li>font name, size or color customization&lt;/li>
+                    &lt;li>justification&lt;/li>
+                    &lt;li>superscript or subscript text&lt;/li>
+                    &lt;li>strikethrough&lt;/li>
+                    &lt;li>underlines&lt;/li>
+                    &lt;li>horizontal rules&lt;/li>
+                &lt;/ul>
+            &lt;/div>
+        </pre>
+    </section>
+    <section id="contenteditable-disabled">
+        <h2>
+            <code>contenteditable-disabled</code> DOM Content Attribute.
```

Comment:
HTML content attributes should first be approved by the WHATWG folks, and should be really be defined in HTML (the behavior of this stuff can be defined by this spec, however). @annevk and/or @domenic might want to give this a nod of approval. 

We should then make a plan for how to integrate the specs.

Reply:
This attribute name violates https://w3ctag.github.io/design-principles/#casing-rules.

Reply:
And yeah, an issue against HTML with a rough outline of the plan/design would be good.

Reply:
@marcoscaceres  and @annevk thank you for your feedback. [Issue](https://github.com/w3c/editing/issues/257) tracking WHATWG discussion. 
I will remove the hyphen. Although, `contenteditabledisabled` seems lengthy to me.